plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/storage/LibvirtStorageAdaptor.java

```diff
@@ -877,8 +877,8 @@ public boolean deletePhysicalDisk(String uuid, KVMStoragePool pool, Storage.Imag
                 rbd.close(image);
                 r.ioCtxDestroy(io);
 
-                s_logger.debug("Succesfully unprotected and removed any snapshots of " + pool.getSourceDir() + "/" + uuid +
-                        " Continuing to remove the RBD image");
+                s_logger.info("Succesfully unprotected and removed all snapshots " + snaps.size() + " of " + pool.getSourceDir() + "/" + uuid +
```

Comment:
This is going to read something like "Succesfully unprotected and removed all snapshots 5 of /<somepath>/<some uuid> Continuing to remove the RBD image", in the logs. the part "all snapshots # of ..." seems not what you'd want to read. How about "Succesfully unprotected and removed any remaining snapshots (counting " + snaps.size() + ") of " + ...?